pkg/tfvars/openstack/rhcos_image.go

```diff
@@ -15,10 +15,36 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// validateBaseImageProperties warns user if the image properties being used are not supported
+func validateImageProperties(imageProperties map[string]string) {
+	validProperties := map[string][]string{
+		"hw_qemu_guest_agent": {"yes", "no", "true", "false"},
+		"hw_scsi_model":       {"virtio-scsci"},
+		"hw_disk_bus":         {"scsi", "virtio"},
+	}
+
+	for property, value := range imageProperties {
+		found := false
+		if validValues, ok := validProperties[property]; ok {
+			for _, allowedValue := range validValues {
+				if value == allowedValue {
+					found = true
+					break
+				}
+			}
+		}
+		if !found {
+			logrus.Warningln("'"+property+"="+value+"'", "is not a supported image property")
```

Comment:
Our rational was that it is already possible to pass an existing glance image with any properties, and this PR was about allowing this feature in the "preferred" installation flow where the installer manages the image.

Also we didn't want to prevent the user from setting any [valid image properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html), because they may be appropriate in their environment. Our message was "use this property at your own risks".

I've changed the validation as requested and we now recommend to use `clusterOSImage` pointing to an existing image in Glance if they want more control over the properties.